app/models/tag.rb

```diff
@@ -407,6 +407,16 @@ def self.graph_data(limit = 250)
     end
   end
 
+  def self.all_tags_by_popularity
+    not_power_tags = []
+    Tag.all.order('count DESC').each do |tag|
```

Comment:
You can refactor this `each` block to a [select iterator](https://www.rubyguides.com/2019/04/ruby-select-method/). They conserve all the elements of the array that match your condition. It would be something like this:
`not_power_tags = Tag.all.order('count DESC').select { |tag| tag.name.include? ":" }`

What do you think?

This is good work :+1:

Reply:
Sure will do this in next commit .